libqtile/widget/mpris2widget.py

```diff
@@ -86,97 +145,144 @@ async def _config_async(self):
         )
 
         if not subscribe:
-            logger.warning("Unable to add signal receiver for %s.", self.objname)
+            msg = "Unable to add signal receiver for {}.".format(self.objname)
+            logger.warning(msg)
```

Comment:
I think the original form would keep consistency with the rest of the logging. Actually I think flake8 will catch this!

Reply:
I'm confused. I thought we needed the `%s` style for logging.

Reply:
Yep we do. That's what it was before, and this change replaced it with a `"{}".format()` call.

Reply:
Ah. Sorry. I thought this was your suggested change but it's actually because I copied some code from an old commit!